platform_setup.sh

```diff
@@ -1,62 +1,60 @@
-# ---------------------------------------------------------------------------- #
-# Valiant Platform application setup script.
```

Comment:
@kranzky maybe keep the information what this setup script is for

Reply:
Sure, no worries @gretch. I tend to be a bit delete-happy.